runtime/default.policy

```diff
@@ -173,13 +175,20 @@ rules:
   - id: base64_decode
     description: The base64 command was used to decode information
     expression: exec.file.name == "base64" && exec.args_flags in ["d"]
+    tags:
+      allow_autosuppression: 'True'
```

Comment:
@YoannGh can you confirm this is how it works ?

Reply:
This wouldn't have worked because the initial agent implementation was only accepting lowercase `true`. Thanks for spotting and updating this @paulcacheux 
I also updated the agent implementation so that `1`, `t`, `T`, `true`, `TRUE`, `True` all work as well.